models/user/user.go

```diff
@@ -413,6 +415,35 @@ func (u *User) DisplayName() string {
 	return u.Name
 }
 
+// EmailTo returns full name and email.
+func (u *User) EmailTo() string {
+	// we don't deal with utf8 to ascii conversion
+	if u.DisplayName() != strings.Trim(strconv.QuoteToASCII(u.DisplayName()), `"`) {
+		return u.Email
+	}
+	// ok just be sure we don't let the user break things somehow
+	sanitizedDisplayName := strings.ReplaceAll(u.DisplayName(), "\n", "")
```

Comment:
This PR seems fine, although I haven't had a chance to full test it.

I feel like there might be a better way to do the sanitization, as this might miss something. Perhaps instead of a blocklist, an allowlist instead.
If the allowlist is not possible (non-utf8 is already being skipped, so I think the allowlist could be just the basic Latin alphabet?), this could be achieved with a slice and then looping over it to reduce duplication.

That being said, we should probably discuss what characters should be allowed. The basic Latin alphabet is not enough for many folks, and we already limit their options elsewhere, so if we don't have to restrict their options here, then I don't think we necessarily need to.

Reply:
Well the thing is that mail relays behave different and a utf8 non valid ascii in FROM is ok ... but not in TO ... etc

So it's a mess and if there would be a suggestion to a nice lib who do stuff for us would be awesome ...

Anyway i could do a regex with a-Z0-9 instead ...

Reply:
done